test_files/expected/clean_header_field_names_success_2.csv

```diff
@@ -0,0 +1,3 @@
+pds_Product_Observational__pds_Identification_Area<1>__pds_logical_identifier<1>,pds_Product_Observational__pds_Identification_Area<1>__pds_version_id<1>,pds_Product_Observational__pds_Observation_Area<1>__pds_Time_Coordinates<1>__pds_start_date_time<1>,pds_Product_Observational__pds_Observation_Area<1>__pds_Time_Coordinates<1>__pds_stop_date_time<1>,pds_Product_Observational__pds_Observation_Area<1>__pds_Primary_Result_Summary<1>__pds_Science_Facets<1>__pds_wavelength_range<1>,pds_Product_Observational__pds_Observation_Area<1>__pds_Primary_Result_Summary<1>__pds_Science_Facets<2>__pds_wavelength_range<1>
```

Comment:
If these are supposed to be clean field names, why do they still have `<1>`?

Reply:
This file and `clean_header_field_names_success.csv` are actually deprecated, since they were replaced in the tests with their `.txt` counterparts. The `.txt` files are correct, and are actually being tested.

Reply:
OK, then the old files that aren't being used should be deleted...